.github/workflows/release.yml

```diff
@@ -125,6 +125,11 @@ jobs:
             curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
             echo "$HOME/.cargo/bin" >> $GITHUB_PATH
           fi
+      - name: "Install perl-IPC-Cmd on manylinux"
```

Comment:
we need this in both places? Is this preserved across upgrades of cargo-dist?

Reply:
Yes, this section is generated by `dist` based on what we've set in out `dist-workspace.toml`.